rpcs3/Emu/Cell/SPULLVMRecompiler.cpp

```diff
@@ -3482,13 +3483,13 @@ class spu_llvm_recompiler : public spu_recompiler_base, public cpu_translator
 #if defined(ARCH_X64)
 			if (utils::get_tsc_freq() && !(g_cfg.core.spu_loop_detection) && (g_cfg.core.clocks_scale == 100))
 			{
+				const auto timebase = m_ir->CreateLoad(get_type<u64>(), m_ir->CreateIntToPtr(m_ir->getInt64(reinterpret_cast<u64>(&g_timebase_offset)), get_type<u64*>()));
```

Comment:
might be nicer to name this timebase_offset

Reply:
Nah too eager to merge the current build lol.